metadata/0xdB3bC9fB1893222d266762e9fF857EB74D75c7D6.json

```diff
@@ -43,7 +43,7 @@
       "Issuer": "ConsolFreight Pilot LLC (Series 4)"
     },
     "discourseLink": "https://discourse.centrifuge.io/t/community-introduction-consolfreight/333",
-    "asset": "Invoice & Cargo",
+    "asset": "Cargo & Freight Forwarding Invoices",
```

Comment:
Changed in discussion with bizdev to better align collateral types of all pools.